wordlistctl.py

```diff
@@ -278,23 +291,45 @@ def main() -> int:
                         help="search local archives")
     search.add_argument("-b", "--base-dir", default=f"{WORDLIST_PATH}", dest="basedir",
                         help="wordlists base directory [default: %(default)s]")
+    search.add_argument("-f", "--fetch", type=int, nargs='+', dest="indexes",
+                        help=(
+                            "fetch the wordlists at the given indexes in the search "
+                            "results, fetch arguments can be used with it"))
+
     search.set_defaults(func=search_func)
 
     lst = subparsers.add_parser("list", help="list wordlists")
     lst.add_argument("-g", "--group",
                      choices=["usernames", "passwords",
                               "discovery", "fuzzing", "misc"],
                      help="group")
+    lst.add_argument("-f", "--fetch", type=int, nargs='+', dest="indexes",
+                        help=(
+                            "fetch the wordlists at the given indexes in the list, "
+                            "fetch arguments can be used with it"))
     lst.set_defaults(func=lst_func)
 
-    results = parser.parse_args()
+    results, fetch_args = parser.parse_known_args()
```

Comment:
Please do not change style unless it's required. Is there a significant advantage of using `parse_known_args`?

Mixing styles will unnecessarily complicate the codebase. [KISS](https://en.wikipedia.org/wiki/KISS_principle) please :)